config/package.xml

```diff
@@ -57,7 +57,7 @@ The goal is to provide an easy-to-use, flexible toolkit that complies with open
   <dependencies>
     <required>
       <php>
-        <min>5.3.10</min>
+        <min>7.2.0</min>
```

Comment:
Why 7.2? I'm not sure if we ever dropped 5.x support on the latest upgrades (@sbreker?), but I think we should at least allow >= 7.0.

Reply:
Argon2 hashing support was added in 7.2 (https://wiki.php.net/rfc/argon2_password_hash) and @djjuhasz said 7.2 was going to be the next version, but maybe we should talk further about about possible issues with this jump.

Reply:
We added support for PHP 7.2 in the AtoM 2.5 Release, in part because it's the default version of PHP installed in Ubuntu 18.04 so it's well tested and known to work.   

PHP 5.6 (the final 5.x release) has been unsupported since Dec. 2018 and PHP 7.1 is only receiving security updates until 1 Dec 2019 (2 days from now).    I think we should drop support for both in the AtoM 2.6 release.

I would like to add support for PHP 7.2 - 7.4 with the AtoM 2.6 release, but I'm not sure whether support for >7.2 will require updates to the code.

Reply:
Refs: 
* (PHP supported versions)[https://www.php.net/supported-versions.php]
* (PHP end-of-life versions)[https://www.php.net/eol.php]